README.md

```diff
@@ -1,138 +1,125 @@
+Registry solution.
 
-### Database related configuration
+## **Standalone**
 
-Some auth configuration is made using environment variables.
-On a Linux system one can set a environment variable with the command
+This service depends on a couple of Python libraries to work. To install them, please run the
+commands below. These have been tested on an Ubuntu 16.04 environment (same used when generating)
+the service's docker image.
 
 ```shell
-  export VAR_NAME=varvalue
+# you may need sudo for those
+apt-get install -y python3-pip
+python3 setup.py
 ```
 
-on a docker-compose schema, one can set environment variables for a container
-Append the following configuration
+You will also need to create and populate the database tables before the first run. In the `python3`
+shell, run:
 
-```shell
-  environment:
-      VAR_NAME: "varvalue"
+```python
+from webRouter import db
+db.create_all()
 ```
 
-The default value is used if the configuration was not provided
-The following variables can be set
+Now create the initial users, groups and permissions:
 
-* AUTH_DB_NAME
-  * database type. Current only postgres is supported
-  * default: postgres
-
-  * AUTH_DB_USER
-    * The username used to access the database
-    * default: auth
-
-  * AUTH_DB_PWD
-        * The password used to access the database
-        * default: empty password
-
-  * AUTH_DB_HOST
-        * The URL used to connect to the database
-        * default: http://postgres
+```shell
+python3 initialConf.py
+```
 
-  * AUTH_KONG_URL
-        * The URL where the Kong service can be found
-        * If set to 'DISABLED' Auth won´t try to configure Kong and will generate secrets for the JWT tokens by itself.
-        * default: http://Kong:8001
+# **Configuration**
 
-  * AUTH_TOKEN_EXP
-        * Expiration time in second for generated JWT tokens
-        * default: 420
+Before running the service, always check if the environment variables are suited to your
+environment.
 
-  * AUTH_TOKEN_CHECK_SIGN
-        * Whether Auth should verify received JWT signatures. Enabling this will cause one extra query to be performed.
-        * default: False
+| Name                  | Description                                                                                                   | Default value   | Accepted values
+| --------------------- | ------------------------------------------------------------------------------------------------------------- | --------------- | ---------------
+| AUTH_CACHE_DATABASE   | Cache database name (or number).                                                                              | 0               | string
+| AUTH_CACHE_HOST       | Address used to connect to the cache.                                                                         | redis           | hostname/IP
+| AUTH_CACHE_NAME       | Type of cache used. Currently only Redis is supported.                                                        | redis           | string
```

Comment:
If set to 'NOCACHE' auth will work without cache. Disabling cache usage considerably degrades performance.